src/components/option-list/OptionList.tsx

```diff
@@ -0,0 +1,146 @@
+  onSelect?: (selectedOption: OptionI) => void;
+}
+
+function OptionList({
+  prevOpt,
+  option,
+  loading,
+  onHoverOption = () => null,
+  onSelect = () => null,
+}: OptionListI) {
+  const [currInd, setCurrInd] = useState(-1);
+  const listRef = useRef<HTMLUListElement>(null);
+
+  useEffect(() => {
+    const listOptions = listRef.current
+      ?.childNodes as unknown as HTMLLIElement[];
+    if (listRef.current) {
+      listOptions.forEach((li: HTMLLIElement, ind: number) => {
+        const handleMouseMove = () => {
+          setCurrInd(ind);
+        };
+        li.addEventListener("mousemove", handleMouseMove);
+      });
+    }
+
+    setCurrInd(-1);
+  }, [prevOpt, option]);
+
+  useEffect(() => {
+    const handelKeyDown = (e: KeyboardEvent) => {
+      let newCurrInd = currInd;
+      if (e.key === "ArrowUp") {
+        newCurrInd =
+          currInd - 1 < 0 ? prevOpt.length + option.length - 1 : currInd - 1;
+      } else if (e.key === "ArrowDown") {
+        newCurrInd =
+          currInd + 1 > prevOpt.length + option.length - 1 ? 0 : currInd + 1;
+      }
+      setCurrInd(newCurrInd);
+    };
+    window.addEventListener("keydown", handelKeyDown);
+    return () => {
+      window.removeEventListener("keydown", handelKeyDown);
+    };
+  }, [currInd, option.length, prevOpt.length]);
+
+  useEffect(() => {
+    if (currInd !== -1) {
+      const listOptions = listRef.current
+        ?.childNodes as unknown as HTMLLIElement[];
+      (
+        listOptions[currInd]?.children[0] as unknown as HTMLButtonElement
+      )?.scrollIntoView({
+        block: "nearest",
+      });
+
+      onHoverOption([...prevOpt, ...option][currInd], currInd);
+    } else onHoverOption(undefined, -1);
+  }, [currInd, onHoverOption, option, prevOpt]);
+
+  return (
+    <ul className={styles.list} ref={listRef}>
+      {prevOpt.map((item, ind) => (
+        <li key={item.value as string}>
+          <Link
+            onClick={() => onSelect(item)}
+            to={`/${item.label}`}
+            className={`${styles.listItem} ${
+              ind === currInd ? styles["listItem--hover"] : ""
+            }`}
+          >
+            <PiClockCounterClockwise size={24} />
+            <span>{item.label}</span>
+            <BsArrowUpRight size={24} />
+          </Link>
+        </li>
+      ))}
+      {loading &&
+        Array(5)
+          .fill(0)
+          .map((item) => (
+            <li
+              className={styles["listItem--skeleton"]}
+              key={`${item + Math.random() * 500}`}
+            >
+              <Skeleton type="circle" width={1.5} height={1.5} />
+              <Skeleton width="100%" height={1} />
+              <Skeleton type="rectangle" width={1.5} height={1.5} />
+            </li>
+          ))}
+
+      {!loading &&
+        option.map((item, ind) => (
+          <li key={item.label + item.value}>
+            <Link
+              onClick={() => onSelect(item)}
+              to={`${item.label}`}
```

Comment:
need to add the '/ ' in  params of to, to avoid above  problem

Reply:
Already fixed this in latest commit of this pr 
https://github.com/chingu-voyages/v46-tier2-team-14/pull/54/commits/c3d13cb7c0d9f36a1714d6d1ff6c183fadaefb57